apps/src/StudioApp.js

```diff
@@ -383,6 +390,7 @@ StudioApp.prototype.init = function(config) {
 
   // Record time at initialization.
   this.initTime = new Date().getTime();
+  this.startTimeMilestone = new Date().getTime();
```

Comment:
nit - would maybe the name milestoneStartTime make slightly more sense? This is the start time of the milestone, not the milestone of the start time, and I think the word order milestoneStartTime communicates that slightly more clearly. In any case its pretty clear what you mean here especially with your good comment above, so I think either name is fine 🙂